tables/heb.tbl

```diff
@@ -0,0 +1,260 @@
+
+# liblouis: Hebrew
+#
+# ------------
+#-index-name: Hebrew uncontracted
+#-display-name: Hebrew uncontracted
+#
+#+language:heb
+#+type:literary
+#+contraction:no
+#+system:heb
+
+#-license: lgpl-2.1
+# This file is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+
+# This file is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+
+# You should have received a copy of the GNU Lesser General Public
+# License along with this file; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+# liblouis  comes with ABSOLUTELY NO WARRANTY.
+
+
+# This table represents Classical or Biblical Hebrew. This table is different from a modern Hebrew table because this table provides support for the points, vowels, cantillation marks, and for the critical apparatus for the BHS.This table only departs from the above mentioned standards, when an innovative braille code is required to represent symbols in the Critical apparatus and cantillation marks, that have never had braille assigned to them before. Modern Hebrew tables do not provide this support. for more information on how the standard for  Biblical Hebrew was developed, please see to the following links::
+#https://en.wikipedia.org/wiki/Hebrew_Braille#:~:text=The%20International%20Hebrew%20Braille%20Code
+#also see: 
+#https://huc.edu/library_blog/learning-about-hebrew-braille/#:~:text=The%20Code%20created%20by%20Brevis,books%20written%20in%20Hebrew%20Braille.
+
+#This braille code was developed by a group of both blind and sighted scholars in an academic setting.
+#-maintainers: Paul Geoghegan <contact@envisionly.tech> and Matityhau Yeshurun <yeatersink@gmail.com>
+letter שְּׁ 146-3
+letter שְּׂ 5-156-3
+letter פְּ 5-1234-3
+letter תְּ 2345-3
+letter לְּ 5-123-3
+letter יְּ 5-245-3
+letter מְּ 5-134-3
+letter נְּ 5-1345-3
+letter ּוְ 136-3
+letter דְּ 5-145-3
+letter קְּ 5-12345-3
+letter שְׁ 146-3
+letter בְּ 12-3
+letter שּׂ 5-156
+letter שְׂ 156-3
+letter שּׁ 146-3
+letter כְּ 13-3
+letter בְּ 12-3
+letter כּ 13
+letter לּ 5-123
+letter יּ 5-245
+letter מּ 5-134
+letter ִי 35
+letter נּ 5-1345
+letter ֵי 3456
+letter ּו 136
+letter וּ 136
+letter וֹ 246
+letter פּ 5-1234
+letter הּ 125
+letter קּ 5-12345
+letter שׁ 146
+letter דּ 5-145
+letter בּ 12
+letter שׂ 156
+letter כְּ 13-3
+letter תְּ 5-1456-3
+letter תּ 2345
+letter ✙ 23456
+letter ❑ 12345
+letter ✕ 1235
+letter ✔ 12356
+letter ❦ 126-126-135-156-1234-345-0-126-34-135-156-1234-345
+letter ❥ 345-126-135-156-1234-345-0-126-34-135-156-1234-345
```

Comment:
I'm just very curious what this sideways-heart symbol is doing in this biblical Hebrew table, and what this 12-character braille representation is telling us.